proto/osmosis/gamm/pool-models/balancer/tx/tx.proto

```diff
@@ -29,3 +33,34 @@ message MsgCreateBalancerPool {
 message MsgCreateBalancerPoolResponse {
   uint64 pool_id = 1 [ (gogoproto.customname) = "PoolID" ];
 }
+
+// ===================== MsgMigrateSharesToFullRangeConcentratedPosition
+message MsgMigrateSharesToFullRangeConcentratedPosition {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+
+  cosmos.base.v1beta1.Coin shares_to_migrate = 2 [
+    (gogoproto.moretags) = "yaml:\"shares_to_migrate\"",
+    (gogoproto.nullable) = false
+  ];
+  // temporary field, eventually gamm pool should be linked to cl pool
+  uint64 pool_id_entering = 3;
+}
```

Comment:
Note to reviewer: Thinking of having some gamm module parameter that stores the canonical cl pool link to its respective gamm pool. At time of upgrade we can set some of the major pools, but other pools will need to be added via governance prop.

Reply:
Acknowledged for now!